build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
     //trick: for the same plugin versions in all sub-modules
-    id("com.android.application").version("8.1.2").apply(false)
-    kotlin("android").version("1.9.20-RC").apply(false)
+    id("com.android.application").version("8.2.0").apply(false)
+    kotlin("android").version("2.0.0").apply(false)
```

Comment:
These should probably just use the version catalog as well, similar to https://github.com/Kotlin/KMP-App-Template/blob/main/build.gradle.kts